release-notes/release-notes57.md

```diff
@@ -83,6 +84,12 @@ For information on:
 - The `spellchecker_ignore_list` option, see: [Spell Checker Pro - `spellchecker_ignore_list`]({{ site.baseurl }}/plugins/premium/tinymcespellchecker/#spellchecker_ignore_list).
 - The `addIgnoredWords` API, see: [Spell Checker Pro - `API`]({{ site.baseurl }}/plugins/premium/tinymcespellchecker/#api).
 
+## Accompanying Premium Skins and Icon Packs changes
+
+The {{site.productname}} 5.7 release includes an accompanying release of the **Premium Skins and Icon Packs** premium content. Included is a new `bootstrap` icon pack that provides [Bootstrap icons](https://icons.getbootstrap.com/) for {{site.productname}}.
```

Comment:
```suggestion
The {{site.productname}} 5.7 release includes an accompanying release of the **Premium Skins and Icon Packs**. Included is a new `bootstrap` icon pack that provides [Bootstrap icons](https://icons.getbootstrap.com/) for {{site.productname}}.
```

Reply:
Hehe, I did originally have that but then added the additional part to keep it consistent with the previous uses: https://www.tiny.cloud/docs/release-notes/release-notes52/#premiumskinsandiconpacks12 (yes this says `premium plugin` which is wrong).

Reply:
:slightly_frowning_face:  that's probably my bad. ignore previous ones, this is better.